devdoc/umock_c_requirements.md

```diff
@@ -24,8 +24,8 @@ It exposes a set of macros and APIs that allow:
 
     typedef void(*ON_UMOCK_C_ERROR)(UMOCK_C_ERROR_CODE error_code);
 
-#define IGNORED_PTR_ARG (NULL)
-#define IGNORED_NUM_ARG (0)
+#define IGNORED_ARG (NULL)
```

Comment:
looks like this should go away

Reply:
Removed

---
In reply to: [395805286](https://github.com/Azure/umock-c/pull/152#discussion_r395805286) [](ancestors = 395805286)